tests/test_serialize_sequence.cc

```diff
@@ -50,6 +51,47 @@ TEST_CASE("make_sequence_filename(SequenceName, UniqueId)", "[serialize_sequence
     REQUIRE(make_sequence_filename(SequenceName{}, 0x1234_uid) == "[0000000000001234]");
 }
 
+TEST_CASE("automation sequence", "[serialize_sequence]")
+{
+    std::stringstream ss;
+
+    Sequence seq("test_sequence");
+
+    SECTION("serialize: automation")
+    {
+        seq.set_automation(true);
+
+        ss << seq;
+
+        Sequence deserialize;
+        ss >> deserialize;
+
+        REQUIRE(deserialize.get_automation());
+    }
+
+    SECTION("serialize: default automation")
+    {
+        ss << seq;
+
+        Sequence deserialize;
+        ss >> deserialize;
```

Comment:
I love the tests, but I still think this should not be an operator. Maybe `void load_sequence_parameters(std::istream& stream, Sequence& sequence)`?

Reply:
See comment on `operator>>(std::istream& stream, Sequence& sequence): std::istream&` above.